components/brave_vpn/resources/panel/components/purchase-failed-panel/index.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+
+import { getLocale } from '../../../../../common/locale'
+import * as S from '../styles/style'
+import { AlertCircleIcon } from 'brave-ui/components/icons'
+import ContactSupport from '../contact-support'
+
+function PurchaseFailedPanel () {
+  const [isContactSupportVisible, setContactSupportVisible] = React.useState(false)
+  const closeContactSupport = () => {
+    console.log('isContactSupportVisible', isContactSupportVisible)
+    setContactSupportVisible(false)
+  }
+  const showContactSupport = () => {
+    setContactSupportVisible(!isContactSupportVisible)
+  }
+  return (
+    <S.Box>
+      <S.PanelContent>
+        <S.IconBox>
+          <AlertCircleIcon color='#84889C' />
+        </S.IconBox>
+        <S.Title>{getLocale('braveVpnPurchaseFailed')}</S.Title>
+        {!isContactSupportVisible && <S.ButtonText onClick={showContactSupport}>
+            {getLocale('braveVpnContactSupport')}
+        </S.ButtonText>
+        }
+        {isContactSupportVisible &&
```

Comment:
I think it looks a little strange to put the support contact form underneath the error. Perhaps we do it like the existing contact form instance, and just replace the UI here (`if (isContactSupportVisible) return <ContactSupport ...>`).

Reply:
good idea, thanks, 
https://user-images.githubusercontent.com/2965009/171626124-3414be42-5cf8-423a-a88e-d1417b4c17b8.mp4